EU4toV2/EU4ToV2.vcxproj.filters

```diff
@@ -282,6 +282,16 @@
     <ClCompile Include="Source\EU4World\Provinces\ProvinceModifier.cpp">
       <Filter>EU4World\Provinces</Filter>
     </ClCompile>
+    <ClCompile Include="..\common_items\StringUtils.cpp" />
```

Comment:
Did this (and the .h file) land in the common_items filter in Visual Studio? If not, please place them there.

Reply:
I do not know what filters are.

Reply:
Well, I found out what they are. I do not recall moving stringutils outside common_items filter (they are physically in the common_items directory), so I moved them back.